### PR TITLE
[IMP] Spreadsheets: updating image and doc links

### PR DESCRIPTION
**Pull request by larm-odoo**
Updating the main doc to streamline the links and update the image.  After updating the main doc, a new FIlters doc is needed (Pr coming after this one).

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @brse-odoo - this is a super short PR, and is ready for a first round review. I am adding a new doc about filters specifically (I didn't realize I was oging to add a new doc till this was opened/started- which is why the eits are so minimal).

File: content/applications/productivity/spreadsheet.rst [hunk 1]
```diff
@@ -12,12 +12,12 @@ Spreadsheet
    spreadsheet/functions
    spreadsheet/templates
 
-Spreadsheet lets you organize, analyze, and visualize your data in tabular form. Among others, you
-can:
+Spreadsheets allow users the ability to organize, analyze, and visualize data in a tabular form.
+Various functions that can be performed within a spreadsheet are:
```
Comment (brse-odoo) [inline content/applications/productivity/spreadsheet.rst:16]:
```suggestion
Spreadsheets allow users to organize, analyze, and visualize data by compiling data sets into tables.
Various functions that can be performed within a spreadsheet are:
```

Comment (brse-odoo) [reply content/applications/productivity/spreadsheet.rst:16]: Optional rewording for clarity!

File: content/applications/productivity/spreadsheet.rst [hunk 2]
```diff
@@ -12,12 +12,12 @@ Spreadsheet
...
 
-- :doc:`Insert and link your Odoo data (pivots, graphs, lists, and menus) <spreadsheet/insert>`.
-- :doc:`Use formulas and functions <spreadsheet/functions>`.
-- :doc:`Use default templates or create new ones <spreadsheet/templates>`.
+- Insert and :doc:`link Odoo data <spreadsheet/insert>` (pivots, graphs, lists, and menus) .
```
Comment (brse-odoo) [inline content/applications/productivity/spreadsheet.rst:18]:
```suggestion
- Insert and :doc:`link Odoo data <spreadsheet/insert>` (pivots, graphs, lists, and menus).
```

**Review by brse-odoo — APPROVED** [review]
@larm-odoo I've finished my review of this PR, great job! Approving with very few suggestions for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

File: content/applications/productivity/spreadsheet.rst [hunk 3]
```diff
@@ -12,12 +12,14 @@ Spreadsheet
    spreadsheet/functions
    spreadsheet/templates
 
-Spreadsheet lets you organize, analyze, and visualize your data in tabular form. Among others, you
-can:
+Spreadsheets allow users to organize, analyze, and visualize data by compiling data sets into
```
Comment (ksc-odoo) [inline content/applications/productivity/spreadsheet.rst:15]:
```suggestion
Spreadsheets help organize, analyze, and visualize data by compiling data sets into
```

**Review by ksc-odoo — APPROVED** [review]
Hey there, @larm-odoo -- just finished my Final Review. Only have a small comment that requires your feedback, other than that, this one is ready for Tech Review.

I fully understand, as you've communicated with me (via Discord), that this is the first part of a larger PR to come -- and that Fabrice, himself, personally requested this from you. Taking that into account, I'd say this one is good to go.

However, if I've mistaken that explanation, I'd suggest making this a bit more detailed/complex, and include some relevant use cases (as suggested in the 'Essentials' related email we all received from JUB recently). Also, I'd suggest (somehow) relating this to the 'Documents' application, as not every spreadsheet is built the *exact* same, so that sort of attention to detail may prove beneficial here for the reader.

Other than that, if you have any questions, don't hesitate to reach out. Thanks! 👍

**Comment by larm-odoo** [comment]
Hi @samueljlieber - this is ready for a tech review!

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @larm-odoo 👋 Thank you for your work, however I do have some critical feedback for you–
This changes in this PR, in its current state, are really not worth merging because the improvements are pretty much replacing pronoun use and an image— there really isnt any value added to the doc in this PR. And with this being a BE scope, pronoun use *is okay*. That being said– I'd like to see more value being added in these improvements, or feel free to close this PR if you dont plan on making these improvements yourself

**Pull request closed without merging**